mws/apimws/migrations/0012_auto_20190204_0918.py

```diff
@@ -15,6 +15,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='phplib',
             name='name_next_os',
-            field=models.CharField(blank=True, max_length=150),
+            field=models.CharField(blank=True, default='', max_length=150),
```

Comment:
this doesn't match the model neither so it makemigrations will show a difference. I suggest reverting back this change, changing models.py to:

```
name_next_os = models.CharField(max_length=150, blank=True, default='')
```

and then re-execute makemigrations